src/assets/stylesheets/base.scss

```diff
@@ -1,2 +1,3 @@
-@import 'assets/stylesheets/base_styles/sanitize';
+// @import 'assets/stylesheets/base_styles/sanitize';
```

Comment:
There seems to some styling that comes with sanitize, I propose we just start everything from scratch with reset. thoughts?